src/library.js

```diff
@@ -992,15 +992,15 @@ addToLibrary({
       /* day of year */     'j': '00[1-9]|0?[1-9](?!\\d)|0?[1-9]\\d(?!\\d)|[1,2]\\d\\d|3[0-6]\\d',
       /* month */           'm': '0[1-9]|[1-9](?!\\d)|10|11|12',
       /* minutes */         'M': '0\\d|\\d(?!\\d)|[1-5]\\d',
-      /* whitespace */      'n': '\\s',
+      /* whitespace */      'n': '\\s*',
```

Comment:
Since you have that pass below anyway, these could probably just be `'n': ' '` and `'t': ' '` as slight simplification. Not sure if it matters though.